src/py-opentimelineio/opentimelineio-bindings/otio_serializableObjects.cpp

```diff
@@ -396,7 +408,11 @@ static void define_items_and_compositions(py::module m) {
              "markers"_a = py::none(),
              py::arg_v("metadata"_a = py::none()));
 
-    py::class_<Clip, Item, managing_ptr<Clip>>(m, "Clip", py::dynamic_attr())
+    py::class_<Clip, Item, managing_ptr<Clip>>(m, "Clip", py::dynamic_attr(), R"docstring(
+The base editable object in OTIO.
+
+Contains a :class:`.MediaReference` and a trim on that media reference.
```

Comment:
Is the "." before MediaReference meant to be a "~"? I'm not familiar with this syntax but I see most of the others are "~"

Reply:
If you are curious, here is two links explaining cross-references in Sphinx:
* https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-syntax
* https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects

Sometimes it's necessary to use `.Something` and sometimes `~Something` is enough.